### PR TITLE
RPP Box Filter on HOST

### PR DESCRIPTION
**Pull request by r-abishek**
- Updates version to RPP 1.9.4
- Adds tensor style box filter on HOST for kernel sizes 3/5/7/9 with U8/F16/F32/I8 bit depths
- NCHW/NHWC variants support
- Adds relevant unit/perf tests

### CODE REVIEW

File: src/modules/cpu/kernel/box_filter.hpp [hunk 1]
```diff
@@ -0,0 +1,2881 @@
...
+    {
+        RpptROI roi;
+        RpptROIPtr roiPtrInput = &roiTensorPtrSrc[batchCount];
+        compute_roi_validation_host(roiPtrInput, &roi, &roiDefault, roiType);
+
+        T *srcPtrImage, *dstPtrImage;
+        srcPtrImage = srcPtr + batchCount * srcDescPtr->strides.nStride;
+        dstPtrImage = dstPtr + batchCount * dstDescPtr->strides.nStride;
+
+        Rpp32u padLength = kernelSize / 2;
```
Comment (rrawther) [inline src/modules/cpu/kernel/box_filter.hpp:318]: change to >>1

File: utilities/test_suite/HOST/Tensor_host.cpp [hunk 2]
```diff
@@ -64,7 +64,8 @@ int main(int argc, char **argv)
     int decoderType = atoi(argv[13]);
     int batchSize = atoi(argv[14]);
 
-    bool additionalParamCase = (testCase == 8 || testCase == 21 || testCase == 23 || testCase == 24 || testCase == 79);
+    bool additionalParamCase = (testCase == 8 || testCase == 21 || testCase == 23 || testCase == 24 || testCase == 49 || testCase == 79);
```
Comment (rrawther) [inline utilities/test_suite/HOST/Tensor_host.cpp:67]: we need to remove all these hardcoded testCase manipulation and use more standard coding methods like string or map table. I guess this should be done in a separate PR

**Review by rrawther — APPROVED** [review]
@r-abishek : can't verify the SSE/AVX code correctness. I am assuming the tests are comparing reference C++ implementation for correctness

**Comment by rrawther** [comment]
@kiritigowda Can you please review this PR and merge if this is OK

**Comment by r-abishek** [comment]
All Jenkins CI passing. Waiting on Azure - https://dev.azure.com/ROCm-CI/ROCm-CI/_build/results?buildId=10400&view=results

**Review by rrawther — APPROVED** [review]

**Review by randyh62 — APPROVED** [review]
Looks good to me.

**Review by LakshmiKumar23 — APPROVED** [review]
Test case 49 passes for both HIP and HOST

**Comment by r-abishek** [comment]
@rrawther @LakshmiKumar23 Mel-Filter Bank and Spectrogram to be ideally merged before Box Filter for correct versioning order.

**Comment by r-abishek** [comment]
@rrawther All 13 checks passing for this PR now
